cloud/jenkins/pg_containers_docker_build.groovy

```diff
@@ -1,7 +1,7 @@
 void build(String IMAGE_POSTFIX){
     sh """
         cd ./source/
-        for PG_VER in 14 13 12; do
+        for PG_VER in 15 14 13 12; do
             docker build --no-cache --squash --build-arg PG_MAJOR=\${PG_VER} \
                 -t perconalab/percona-postgresql-operator:${GIT_PD_BRANCH}-ppg\${PG_VER}-${IMAGE_POSTFIX} \
```

Comment:
Do we have a rule to support only 3 versions?

Reply:
no